src/test/java/studyconnect/StudyConnectApplicationTest.java

```diff
@@ -0,0 +1,16 @@
+package studyconnect;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.studyconnect.StudyConnectApplication;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.studyconnect.StudyConnectApplication' import. Should be before 'org.springframework.boot.test.context.SpringBootTest'.